apps/cow-fi/data/cow-amm/const.tsx

```diff
@@ -174,12 +174,7 @@ export const FAQ_DATA = [
           href="https://cowprotocol.typeform.com/cow-amm-lpers"
           external
           utmContent="cow-amm-contact-us"
-          onClick={() =>
-            sendEventHandler({
-              category: EventCategories.COWAMM,
-              action: 'Content link click - FAQ:Contact us',
-            })
-          }
```

Comment:
In general, all event triggering has been to a `events.ts` file per app that has all analytics events and abstract about the analytics implementation details